core/src/main/codegen/templates/Parser.jj

```diff
@@ -4313,6 +4316,11 @@ SqlNode StringLiteral() :
     // or comments may not occur between the prefix and the first quote, the
     // head fragment, with any prefix, is one token.
 
+<#if parser.allowHexCharacterLiteral>
+    hexCharLiteral = SqlHexCharStringLiteral()
+    {return hexCharLiteral;}
+    |
```

Comment:
nit: I think this "|" should be shifted all the way to the left.

Reply:
Following the [convention](https://github.com/googleinterns/calcite/blob/d76a6ac26a8bc662234e4e0b57c5d11f384d3c05/core/src/main/codegen/templates/Parser.jj#L4357) in the same method, this "|" could be at the right indentation.